front/src/common/BootstrapSNCF/SwitchSNCF/SwitchSNCF.tsx

```diff
@@ -8,7 +8,7 @@ export const SWITCH_TYPES = {
   switch: 'switch',
 };
 
-type Props = {
+export type SwitchSNCFProps = {
```

Comment:
No need to export this type only for `onChange: InputHTMLAttributes<HTMLInputElement>['onChange'];`
```suggestion
type SwitchSNCFProps = {
```

Reply:
So if in a futur we change the definition of this `onChange` function (like something `(e:boolean)=>void` instead of the classic HTML onChange),  the code will not break. 
I really prefer to take the definition of the component instead of the generic html one.
And there is no cost to export a type.

Reply:
Ok I understand